pkg/splunk/controller/statefulset.go

```diff
@@ -317,16 +317,22 @@ func isCurrentCROwner(cr splcommon.MetaObject, currentOwners []metav1.OwnerRefer
 	return reflect.DeepEqual(currentOwners[0], splcommon.AsOwner(cr, false))
 }
 
-// IsStatefulSetScalingUp checks if we are currently scaling up
-func IsStatefulSetScalingUp(client splcommon.ControllerClient, cr splcommon.MetaObject, name string, desiredReplicas int32) (bool, error) {
+// IsStatefulSetScalingUpOrDown checks if we are currently scaling up or down
+func IsStatefulSetScalingUpOrDown(client splcommon.ControllerClient, cr splcommon.MetaObject, name string, desiredReplicas int32) (bool, bool, error) {
```

Comment:
As discussed can we use a enum to indicate the Scale up/down rather than 2 bools)